docs/pages/reference/backends.mdx

```diff
@@ -1467,3 +1467,92 @@ teleport:
     sync: NORMAL
     journal: WAL
 ```
+
+## CockroachDB
+
+Teleport can use [CockroachDB](https://www.cockroachlabs.com/) as a storage backend
+to achieve high available and survive regional failures. You must take steps to 
+protect access to CockroachDB in this configuration because that is where Teleport
+secrets like keys and user records will be stored.
+
+<Admonition
+  type="warning"
+  title="IMPORTANT"
+>
+  `CockroachDB` can currently only be used to store Teleport's internal database in a
+  highly-available way. This will allow you to have multiple auth servers in your
+  cluster for a high availability deployment, but it will not also store Teleport audit events
+  for you in the same way that [DynamoDB](#dynamodb) or
+  [Firestore](#firestore) will.
+</Admonition>
+
+At a minimum you must configure CockroachDB to allow Teleport to create tables.
+Teleport will create the database if given permission to do so but this is not
+required if the database already exists.
+
+```
+CREATE DATABASE database_name;
+CREATE USER database_user;
+GRANT CREATE ON DATABASE database_name TO database_user;
+```
+
+You must also enable change feeds in CockroachDB's cluster settings. Teleport
+will configure this setting itself if granted `SYSTEM MODIFYCLUSTERSETTING`.
+
+```
+SET CLUSTER SETTING kv.rangefeed.enabled = true;
+```
+
+For more details on deploying CockroachDB see [CockroachDB's deployment options](https://www.cockroachlabs.com/docs/stable/choose-a-deployment-option).
+
+For more details on configuring multi-region survival goals in CockroachDB see
```

Comment:
"For more details" suggests that there are at least some details here, but I don't see anything about configuring multi-region survival goals.

I would rephrase this. Perhaps something along the lines of this:

> There are several ways to configure multi-region deployments of CockroachDB,
> the details of which are not in scope for this guide. To learn about how to
> configure multi-region survival goals, see ...

Reply:
Makes sense I updated this for both the deployment and survival goals here https://github.com/gravitational/teleport/pull/37880/commits/b42399a329254ca6d30e7cbfdef2599b8ca8cb58